tardis/plasma/properties/nlte.py

```diff
@@ -14,4 +21,85 @@ class PreviousElectronDensities(PreviousIterationProperty):
     outputs = ('previous_electron_densities',)
 
 class PreviousBetaSobolevs(PreviousIterationProperty):
-    outputs = ('previous_beta_sobolevs',)
\ No newline at end of file
+    outputs = ('previous_beta_sobolevs',)
+
+class HeliumNLTE(ProcessingPlasmaProperty):
+    outputs = ('helium_population',)
+
+    def calculate(self, level_boltzmann_factor, electron_densities,
+        ionization_data, beta_rad, g, g_electron, w, t_rad, t_electrons,
+        delta, zeta_data, number_density, partition_function):
+        helium_population = level_boltzmann_factor.ix[2].copy()
+        # He I excited states
+        he_one_population = self.calculate_helium_one(g_electron, beta_rad,
+            partition_function, ionization_data, level_boltzmann_factor,
+            electron_densities, g, w, t_rad, t_electrons)
+        helium_population.ix[0].update(he_one_population)
+        #He I metastable states
+        helium_population.ix[0].ix[1] *= (1 / w)
+        helium_population.ix[0].ix[2] *= (1 / w)
+        #He I ground state
+        helium_population.ix[0].ix[0] = 0.0
+        #He II excited states
+        he_two_population = level_boltzmann_factor.ix[2,1].mul(
+            (g.ix[2,1].ix[0]**(-1)) * (t_electrons / t_rad)**0.5)
+        helium_population.ix[1].update(he_two_population)
+        #He II ground state
+        helium_population.ix[1].ix[0] = 1.0
+        #He III states
+        helium_population.ix[2].ix[0] = self.calculate_helium_three(t_rad, w,
+            zeta_data, t_electrons, delta, g_electron, beta_rad,
+            partition_function, ionization_data, electron_densities)
+        unnormalised = helium_population.sum()
+        normalised = helium_population.mul(number_density.ix[2] / unnormalised)
+        helium_population.update(normalised)
+        return helium_population
+
+    def calculate_helium_one(self, g_electron, beta_rad, partition_function,
+            ionization_data, level_boltzmann_factor, electron_densities, g,
+            w, t_rad, t_electron):
+        (partition_function_index, ionization_data_index, partition_function,
+            ionization_data) = self.filter_with_helium_index(2, 1,
+            partition_function, ionization_data)
+        phis = (1 / PhiSahaLTE.calculate(g_electron, beta_rad,
+            partition_function, ionization_data)) * electron_densities * \
+            (1.0/g.ix[2].ix[1].ix[0]) * (1/w) * (t_rad/t_electron)**(0.5)
```

Comment:
Changed to .ix[2,1,0]. I used to use .iloc but it doesn't work with one of the Pandas versions on here so I always use .ix now.